### PR TITLE
CB-755 AWS owner tag contains the user id instead of the human readable user name

### PR DESCRIPTION
**Pull request by daszabo**

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/stack/provision/action/AbstractStackCreationAction.java [hunk 1]
```diff
@@ -45,8 +45,8 @@ protected StackContext createFlowContext(String flowId, StateContext<StackCreati
         Stack stack = stackService.getByIdWithListsInTransaction(payload.getStackId());
         MDCBuilder.buildMdcContext(stack);
         Location location = location(region(stack.getRegion()), availabilityZone(stack.getAvailabilityZone()));
-        CloudContext cloudContext = new CloudContext(stack.getId(), stack.getName(), stack.cloudPlatform(), stack.getOwner(), stack.getPlatformVariant(),
-                location, stack.getCreator().getUserId(), stack.getWorkspace().getId());
+        CloudContext cloudContext = new CloudContext(stack.getId(), stack.getName(), stack.cloudPlatform(), stack.getPlatformVariant(),
+                location, stack.getCreator().getUserId(), stack.getCreator().getUserId(), stack.getWorkspace().getId());
```
Comment (keyki) [inline core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/stack/provision/action/AbstractStackCreationAction.java]: stack.getCreator().getUserId() is used 2 times here, is this intended?

File: cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsResourceConnector.java [hunk 2]
```diff
@@ -422,7 +422,7 @@ private void resumeAutoScaling(AmazonAutoScalingRetryClient amazonASClient, Coll
                 new Parameter().withParameterKey("CBUserData").withParameterValue(stack.getImage().getUserDataByType(InstanceGroupType.CORE)),
                 new Parameter().withParameterKey("CBGateWayUserData").withParameterValue(stack.getImage().getUserDataByType(InstanceGroupType.GATEWAY)),
                 new Parameter().withParameterKey("StackName").withParameterValue(stackName),
-                new Parameter().withParameterKey("StackOwner").withParameterValue(ac.getCloudContext().getOwner()),
+                new Parameter().withParameterKey("StackOwner").withParameterValue(ac.getCloudContext().getUserId()),
```
Comment (keyki) [inline cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsResourceConnector.java]: I thought we need the user name here. Or is the user id the email address here?

Comment (daszabo) [reply cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsResourceConnector.java]: Yes, userid is the user's e-mail address. The owner was the uuid.

Comment (keyki) [reply cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsResourceConnector.java]: cool, thank you

**Pull request merged**